src/main/webapp/dev/js/main/instructorFeedbackEdit.es6

```diff
@@ -1087,7 +1087,11 @@ function readyFeedbackEditPage() {
     setupQuestionCopyModal();
 
     // Additional formatting & bindings.
-    disableEditFS();
+    if ($('#form_feedbacksession').data(`${ParamsNames.FEEDBACK_SESSION_ENABLE_EDIT}`) === true) {
```

Comment:
Oh I see! That changes things. Maybe use `c:if` to remove the entire data attribute if it's not set then.